views/package-lock.json

```
@@ -2769,6 +2765,12 @@
             "integrity": "sha1-qdhPJj7I/obS4WzCz36pq8IA1ho=",
             "requires": {
                 "eve": "git://github.com/adobe-webplatform/eve.git#eef80ed"
+            },
+            "dependencies": {
+                "eve": {
+                    "version": "git://github.com/adobe-webplatform/eve.git#eef80ed8d188423c2272746fb8ae5cc8dad84cb1",
+                    "from": "git://github.com/adobe-webplatform/eve.git#eef80ed"
```

Shouldn't the source-dependency use `http` to allow unauthorized requests?

```suggestion
                "eve": "https://github.com/adobe-webplatform/eve.git#eef80ed"
            },
            "dependencies": {
                "eve": {
                    "version": "https://github.com/adobe-webplatform/eve.git#eef80ed8d188423c2272746fb8ae5cc8dad84cb1",
                    "from": "https://github.com/adobe-webplatform/eve.git#eef80ed"
```

---

I don't know what I did there :/